executor/hash_table.go

```diff
@@ -501,6 +502,10 @@ func (c *hashRowContainer) Len() uint64 {
 }
 
 func (c *hashRowContainer) Close() error {
+	failpoint.Inject("issue60923", func() {
```

Comment:
You can use `failpoint.Inject("issue60923", nil)` and `failpoint.Enable("github.com/pingcap/tidb/executor/issue60923", "panic")`